user-access-service/src/main/java/no/unit/nva/database/DatabaseSubService.java

```diff
@@ -13,10 +13,13 @@
 import no.unit.nva.useraccessmanagement.model.interfaces.Validable;
 import nva.commons.core.JsonSerializable;
 import nva.commons.core.attempt.Failure;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class DatabaseSubService {
```

Comment:
This is supposed to be an abstract class, but PMD does not allow abstract classes without abstract methods